apps/gro/fields/email-address.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  'email-text': {
+    mixin: 'input-text',
```

Comment:
couldn't this be an email mixin instead, then you wouldn't need to invoke the email validator